MessengerOnFileForLinux/DisplayChat/src/InformationPrinter.hpp

```diff
@@ -5,5 +5,8 @@
 class InformationPrinter
 {
 public:
-    void printInformation(const std::string&) const;
+    virtual void printInformation(const std::string&) const;
```

Comment:
Jeżeli te virtuale są tylko do stubów to zrób klasę interfejs i klasę, która dziedziczy i to używa oraz zmień stuba tak by dziedziczył po interfejsi --> wydaje mi się, że jest jakiś pattern na to. Warto poszukać by zrobić to dobrze i ładnie, ja nie jestem pewny czy dobrze Ci powiedziałem.